index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <h1>Hello Microverse</h1>
+  </body>
```

Comment:
As requested in the project requirements:
![image](https://user-images.githubusercontent.com/42799579/204864177-110e95bc-2736-48e6-834a-54839847659a.png)

Please make sure to add a class to your `h1`, and style it in your `css` file 😉.